src/mds/layout/Header.jsx

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import { MAX_WIDTH, PATH } from "../../constants";
+import { BrandLogo } from "../icon";
+import { CategoryTabBar } from "../category";
+import { RowBox } from "../box";
+import { TextButton } from "../button";
+import { useNavigate } from "react-router-dom";
+import Divider from "../Divider";
+
+const Header = () => {
+  const navigate = useNavigate();
+  const { memberId } = { ...localStorage }; // FIXME :  will be replaced by using hooks.
+
+  const handleMainPage = () => navigate(PATH.MAIN);
+  const handleAlarmPage = () => navigate(PATH.ALARM(memberId));
+  const handleSignInPage = () => navigate(PATH.SINGIN);
+  const handleProPage = () => navigate(PATH.PRO_CENTER);
+
+  return (
+    <>
+      <Container>
+        <RowBox height={"3rem"} justifyContent={"space-between"}>
+          <BrandLogo onClick={handleMainPage} />
+
+          {/* TODO: To be add Seaarch Bar when fix design and interactions */}
+
+          <TextButtonGroup>
```

Comment:
```suggestion
          <TextButtonContainer>
```

`Container`와 `Group`의 차이점은 어떤게 있을까요?

Reply:
Container 는 태그 자체에는 의미가 없으며 단순히 요소를 묶기 위한 용도로 사용하였고 , Group 은 비슷한 성질끼리 묶기 위한 용도로 사용했었습니다 !